unittests/data/BackendInterfaceTests.cpp

```diff
@@ -72,3 +77,41 @@ TEST_F(BackendInterfaceTest, FetchFeesLegacySuccessPath)
         EXPECT_EQ(fees->reserve, XRPAmount(3));
     });
 }
+
+TEST_F(BackendInterfaceTest, FetchLedgerPageSuccessPath)
+{
+    using namespace ripple;
+
+    backend->setRange(MINSEQ, MAXSEQ);
+    EXPECT_FALSE(backend->cache().isDisabled());
+    EXPECT_CALL(*backend, doFetchSuccessorKey(_, _, _))
+        .Times(10)
+        .WillRepeatedly(Return(uint256{"1FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF1FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"}));
+    EXPECT_CALL(*backend, doFetchLedgerObjects(_, _, _))
+        .WillOnce(Return(std::vector<Blob>{
+            Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}, Blob{'s'}
```

Comment:
`std::vector<Blob>{10, Blob{'s'}}` would look a bit nicer

Reply:
Agree 👍 copy-pasting is more fun though 😃